app/src/main/res/values-fr/strings.xml

```diff
@@ -100,11 +100,11 @@
     <string name="stardust_cost">Coût en Poussière</string>
     <string name="evolution">Évolution</string>
     <string name="level">Niveau</string>
-    <string name="level_num">Niveau %1$.1f</string>
+    <string name="level_num">Niveau %.1f</string>
     <string name="refine">Raffiner</string>
     <string name="refine_no_progress">Le raffinement n\'a pas amélioré les résultats</string>
-    <string name="last_scan">Dernier scan : %1$s</string>
```

Comment:
In French there is a space both before and after a colon. There's also a space before lots of punctuation marks like ! ? and ;